src/Apps/Order/Routes/Shipping/Components/__tests__/SavedAddresses.jest.tsx

```diff
@@ -240,26 +240,29 @@ describe("Saved Addresses", () => {
         phoneNumber: "555-555-5555",
       }
       const addAddressButton = screen.getByText("Add a new address")
-      await userEvent.click(addAddressButton)
+      userEvent.click(addAddressButton)
 
       screen.getByText("Add address")
 
       await fillAddressFormFields(validAddress)
 
-      await flushPromiseQueue()
-
       mockExecuteUserAddressAction.mockResolvedValueOnce({
         data: { ...validAddress },
       })
 
-      await userEvent.click(screen.getByText("Save"))
+      await waitFor(
+        () => {
+          userEvent.click(screen.getByText("Save"))
+        },
+        // Bottleneck waiting for form updates
+        { timeout: 4000 }
```

Comment:
Might as well bump this more vs erroring out PR?

Reply:
It was actually the overall test timeout limit of 10s that failed. I just increased it to see what we are working with here.